artemis/domains.py

```diff
@@ -1,3 +1,20 @@
+from publicsuffixlist import PublicSuffixList
+
+PUBLIC_SUFFIX_LIST = PublicSuffixList()
+
+
+def is_main_domain(domain: str) -> bool:
```

Comment:
Isn't this basically `return PUBLIC_SUFFIX_LIST.privatesuffix(domain) == domain`?
```python
>>> psl.privatesuffix("example.com")
'example.com'
>>> psl.privatesuffix("aaaa.example.com")
'example.com'
>>> psl.privatesuffix("google.co.uk")
'google.co.uk'
>>> psl.privatesuffix("a.b.c.google.co.uk")
'google.co.uk'
```